src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -71,4 +89,95 @@ public enum JsonTokenType : byte
         public bool TryGetInt64Value(out long value) { throw null; }
         public bool TryGetSingleValue(out float value) { throw null; }
     }
+    public ref partial struct Utf8JsonWriter
+    {
+        private object _dummy;
+        public Utf8JsonWriter(System.Buffers.IBufferWriter<byte> bufferWriter, System.Text.Json.JsonWriterState state = default(System.Text.Json.JsonWriterState)) { throw null; }
+        public long BytesCommitted { get { throw null; } }
+        public long BytesWritten { get { throw null; } }
+        public int CurrentDepth { get { throw null; } }
+        public System.Text.Json.JsonWriterState CurrentState { get { throw null; } }
+        public void Flush(bool isFinalBlock = true) { }
+        public void WriteBoolean(System.ReadOnlySpan<byte> propertyName, bool value, bool suppressEscaping = false) { }
+        public void WriteBoolean(System.ReadOnlySpan<char> propertyName, bool value, bool suppressEscaping = false) { }
+        public void WriteBoolean(string propertyName, bool value, bool suppressEscaping = false) { }
+        public void WriteBooleanValue(bool value) { }
+        public void WriteCommentValue(System.ReadOnlySpan<byte> utf8Text, bool suppressEscaping = false) { }
+        public void WriteCommentValue(System.ReadOnlySpan<char> utf16Text, bool suppressEscaping = false) { }
+        public void WriteCommentValue(string utf16Text, bool suppressEscaping = false) { }
```

Comment:
There are a bunch of places in the API where we're taking `Readonlyspan<byte>`, `Readonlyspan<char>`, and `string` in respective overloads, but here we're explicit about them being "utf8text" and "utf16text", whereas elsewhere we just say "propertyName" or "value".  Why the difference?

Reply:
Leftover artifact from when I had different API overloads (like WriteRaw, WriteAsBase64, etc.) where the encoding was important for clarity. We don't need it here (or in WriteStringValue). Will fix accordingly.